client/flutter/pubspec.yaml

```diff
@@ -49,6 +49,7 @@ dependencies:
   path_provider: ^1.6.5
   fl_chart: ^0.8.7
   s2geometry: 0.1.1
+  flutter_secure_storage: 3.3.3
```

Comment:
https://github.com/mogol/flutter_secure_storage/issues/69 
This package has several cryptography issues that have been ignored since July/August 2019 and apparently it crashes on some devices
https://github.com/mogol/flutter_secure_storage/issues

Reply:
Good to know!

Reply:
Maybe it's worth forking and cleaning up the package then auditing it with some security folks?